cobalt/build/configs/linux-x64x11-evergreen/args.gn

```diff
@@ -1,18 +1,3 @@
-import("//cobalt/build/configs/linux_common.gn")
+import("//cobalt/build/configs/linux-x64x11-modular/args.gn")
 
-use_custom_libc = true
-
-# TODO: b/384652502 - Cobalt: Stop overriding enable_pkeys
-# ../../base/allocator/partition_allocator/pkey.cc:35:18: error: use of undeclared identifier 'SYS_pkey_mprotect'
-#   return syscall(SYS_pkey_mprotect, addr, len, prot, pkey);
-enable_pkeys = false
-
-# We don't implement the necessary APIs for mdns.
-enable_mdns = false
-
-# This brings in libraries we can't have in hermetic builds.
-use_system_libdrm = false
-
-# This brings in libraries we can't have in hermetic builds, and our decoding
-# is done under Starboard, so we can disable decoders at the Chromium level.
-rtc_use_h264 = false
+is_evergreen = true
```

Comment:
I prefer use_evergreen as the cobalt-level targets are built mostly the same way, it mostly indicates a dependency on the loader_app/elf_loader_sandbox vs the *_loader targets. Seems more in line with the GN style guide.